<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udp_modem_widget</class>
 <widget class="QWidget" name="udp_modem_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP波形发生器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget_display" native="true">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0" colspan="3">
          <widget class="QTableWidget" name="tableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>810</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTextEdit" name="textEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>数据格式配置</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>数据字长： </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox_word_len"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>数据类型：</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="comboBox_data_iq"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>110</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>控制区</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>开始</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>暂停</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>清除</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>更新</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_config" native="true">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>发生器状态配置</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>本机地址：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_local_ip"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>本机端口：</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_local_port"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>对端地址：</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_dest_ip"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>对端端口：</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="lineEdit_dest_port"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_channel_config" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_channel_config">
              <property name="title">
               <string>通道状态配置</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="2" column="0">
                <widget class="QCheckBox" name="checkBox_ch3">
                 <property name="text">
                  <string>通道3</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QCheckBox" name="checkBox_ch6">
                 <property name="text">
                  <string>通道6</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QCheckBox" name="checkBox_all_channel_on">
                 <property name="text">
                  <string>All ON/OFF</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_ch1">
                 <property name="text">
                  <string>通道1</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QCheckBox" name="checkBox_ch4">
                 <property name="text">
                  <string>通道4</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QCheckBox" name="checkBox_ch5">
                 <property name="text">
                  <string>通道5</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBox_ch2">
                 <property name="text">
                  <string>通道2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>波形参数配置</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="8" column="1" colspan="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>噪声功率：</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1" colspan="2">
                <widget class="QComboBox" name="comboBox_4"/>
               </item>
               <item row="6" column="1" colspan="2">
                <widget class="QSpinBox" name="spinBox"/>
               </item>
               <item row="5" column="1" colspan="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>波形间隔：</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QCheckBox" name="checkBox_freqset">
                 <property name="text">
                  <string>步进锁</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1" colspan="2">
                <widget class="QComboBox" name="comboBox_5"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>波形类型：</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>初始延迟：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>平均功率：</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>载波频率：</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="2">
                <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>符号速率：</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1" colspan="2">
                <widget class="QSpinBox" name="spinBox_2"/>
               </item>
               <item row="1" column="1">
                <widget class="FreqSpinBox" name="doubleSpinBox_carrier_freq"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>波形参数1：</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="2">
                <widget class="QComboBox" name="comboBox_3"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>采样速率：</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FreqSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">freqspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
